packages/oui-button/protractor.conf.js

```diff
@@ -0,0 +1,8 @@
+import fs from 'fs'
```

Comment:
I don't think we will use Protractor for now. So you can remove it.

Reply:
done